### PR TITLE
DIGI 15 - Create Tax Page

### PR DESCRIPTION
**Pull request by anebhargavkrishna**
1.Added TaxpageDataModel to access data from data base
2.Added TaxpageController

### CODE REVIEW

File: DigiOrderApi/DigiOrderApi/Controllers/TaxpageController.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            try
+            {
+                dynamic json = data;
+                taxPageDataModel = new TaxpageDataModel();
+                dtRecords = taxPageDataModel.PostTaxPage(Convert.ToString(json.Title), Convert.ToString(json.Description));
+                dtRecords.TableName = "TAXPAGES";
+            }
+            catch (Exception ex)
+            {
+                return Request.CreateErrorResponse(HttpStatusCode.NotFound, "Record Not Found");
```
Comment (RAMUNIMMALA) [inline DigiOrderApi/DigiOrderApi/Controllers/TaxpageController.cs:40]: provide correct status code as no records are verified in the endpoint

File: DigiOrderApi/DigiOrderApi/Controllers/TaxpageController.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+
+namespace DigiOrderApi.Controllers
+{
+    public class TaxpageController : ApiController
+    {
+        TaxpageDataModel taxPageDataModel = null;
```
Comment (RAMUNIMMALA) [inline DigiOrderApi/DigiOrderApi/Controllers/TaxpageController.cs:14]: First letter should be in upper case if it not a type prefix

File: DigiOrderApi/DigiOrderApi/Controllers/TaxpageController.cs (hunk 1 above)
Comment (anebhargavkrishna) [reply DigiOrderApi/DigiOrderApi/Controllers/TaxpageController.cs:40]: added proper message

File: DigiOrderApi/DigiOrderApi/Controllers/TaxpageController.cs (hunk 2 above)
Comment (anebhargavkrishna) [reply DigiOrderApi/DigiOrderApi/Controllers/TaxpageController.cs:14]: Changed firt letter to lower to upper case

**Pull request merged**